oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -581,6 +601,15 @@ static int mca_spml_ucx_ctx_create_common(long options, mca_spml_ucx_ctx_t **ucx
         goto error;
     }
 
+    if (mca_spml_ucx.synchronized_quiet) {
+        ucx_ctx->put_op_mask = calloc(SSHMEM_MASK_LEN(nprocs), SSHMEM_MASK_SIZE);
```

Comment:
why do we allocate the mask in 2 places?

Reply:
there are 2 locations where context is initialized. shifted put-op initialization into separate function